docs/01_cva6_user/CVX_Interface_Coprocessor.rst

```diff
@@ -23,12 +23,249 @@ CV-X-IF Interface and Coprocessor
+                                         │     issue if  │ coprocessor ├──┘
+                                         └──────────────►│             │
+                                                         │             │
+                                                         └─────────────┘
+         if = interface
+
+**Notes**
+
+Only the 3 mandatory interfaces from the specification (issue, commit and result
+) have been implemented.
+Compressed interface, Memory Interface and Memory result interface are not yet
+implemented in the CVA6.
+
+Inputs/Outputs of CVXIF Functional Unit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+==================== ========= ============= ==================================================
+Signal               Direction Connected to  Description
+==================== ========= ============= ==================================================
+**clk_i**            I         ariane input  | clock
+**rst_ni**           I         ariane input  | reset
+**fu_data_i**        I         issue stage   | Data needed to execute the offloaded instruction
+                                             | (operands, operation, FU, trans_id).
+**x_valid_i**        I         issue_stage   | Signal that issue stage wants to use cvxif
+                                             | functional unit
+**x_ready_o**        O         issue stage   | Signal that cvxif functional unit is ready
+                                             | to accept an offloaded instruction
+**x_off_instr_i**    I         issue stage   | Offloaded instruction
+**x_trans_id_o**     O         commit stage  | Scoreboard ID of the instruction to be committed
+**x_exception_o**    O         commit stage  | Data on exception that could have occurred in
+                                             | the coprocessor
+**x_result_o**       O         commit stage  | Result of the operation
+**x_valid_o**        O         commit stage  | Signal that instruction is valid to be committed
+**x_we_o**           O         commit stage  | Signal that a writeback is needed for the result
+                                             | sent by the coprocessor. If we == 0 then
+                                             | scoreboard entry of the instruction will change
+                                             | destination register to reg 0
+**x_cvxif_req_o**    O         copro inputs  | All signals connected to coprocessor inputs. See
+                                             | `CVXIF Functional Unit and writeback` part.
+**x_cvxif_resp_i**   I         copro outputs | All signals connected to coprocessor outputs.
+                                             | See `CVXIF Functional Unit and writeback` part.
+==================== ========= ============= ==================================================
+
+Supported Parameters
+~~~~~~~~~~~~~~~~~~~~
+The following table presents CVXIF parameters supported by CVA6.

```

Comment:
Parameters are needed for the user. You can guide them to select the right values.